packages/ModdableDataGrid/ModeRepository.ts

```diff
@@ -10,9 +10,10 @@ export class ModeRepository<TData, TDataFetcherParameters extends FetchableDataG
 
 	constructor(private readonly dataGrid: ModdableDataGrid<TData, TDataFetcherParameters>) {
 		super(`ModdableDataGrid.${dataGrid.tagName.toLowerCase()}.Modes`, [], (_key: string, value: any) => {
-			return (DateTime.isoRegularExpression.test(value))
-				? new Date(value)
-				: value
+			if (typeof value === 'object' && value !== null && ('start' in value || 'end' in value)) {
```

Comment:
@a11delavar, take a look at the history of commits. That's how I wanted it to look like 

<img width="1075" alt="Screenshot 2024-05-03 at 13 31 17" src="https://github.com/3mo-esolutions/web-components/assets/142882738/c9261250-827f-4a8b-b938-d02770943026"> 

After all, I decided to simplify that.